migration-engine/connectors/mongodb-migration-connector/tests/migrations/tests.rs

```diff
@@ -42,4 +42,9 @@ scenarios! {
   multi_column_mixed_fulltext_indexes_can_be_changed
   pushing_without_preview_flag_should_not_touch_fts_indexes
   unique_to_index
+  composite_indexes_can_be_created
+  composite_indexes_can_be_created_descending
+  composite_indexes_can_be_changed_from_descending_to_ascending
+  composite_indexes_can_be_dropped
+  composite_indexes_can_be_renamed
```

Comment:
Next time I have time to work on that test setup, I want to automate this part with a build script. It's too easy to forget.

Reply:
Yeah, and I felt like copying these files, and adding them here was a bit too much work. And I basically needed to copy every single test, so was better to just add one composite to each of them.